addons/point_of_sale/static/src/app/models/pos_order.js

```diff
@@ -86,10 +86,12 @@ export class PosOrder extends Base {
         return this.state !== "draft";
     }
 
-    getOrderName() {
-        return this.getFloatingOrderName() || "";
+    getGeneralNote() {
+        return this.general_note || "";
+    }
+    setGeneralNote(general_note) {
+        this.general_note = general_note;
```

Comment:
let's not have this setter and getter.

During `setup` in this class, make sure that that `general_note` defaults to empty string value.

And if you want to access its value, just do `<order obj>.general_note`.